docs/storage/volume_migration.md

```diff
@@ -0,0 +1,222 @@
+# Migration update volume strategy and volume migration
+
+Storage migration is possible while the VM is running by using the update volume strategy. Storage migration can be useful in the cases where the users need to change the underlying storage, for example, if the storage class has been deprecated, or there is a new more performant driver available.
+
+This feature doesn't handle the volume creation or cover migration between storage classes, but rather implement a basic API which can be used by overlaying tools to perform more advanced migration planning.
+
+If `Migration` is specified as `updateVolumesStrategy`, KubeVirt will try to migrate the storage from the old volume set to the new one when the VirtualMachine spec is updated. The migration considers the changed volumes present into a single update. A single update may contain modifications to more than one volume, but sequential changes to the volume set will be handled as separate migrations.
```

Comment:
It might make sense to describe how users of this feature can follow the status of the migration?

Reply:
Yes, I have added how to watch the migration using the label